shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -220,7 +262,10 @@ public static native void nativeOnVsync(
 
   // ----- End Engine FlutterTextUtils Methods ----
 
-  @Nullable private Long nativePlatformViewId;
```

Comment:
This thing was a misnomer. The receiving side file is named platform view but is really a Shell (which owns the platform view). Renamed to reduce confusion.